themes/default/content/docs/guides/crosswalk/aws/iam.md

```diff
@@ -56,7 +56,9 @@ OR across all of those policies when evaluating them.
 For more extensive details about IAM policies and their contents, refer to the [AWS access policies documentation](
 https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html).
 
-### Using the PolicyDocument Interface
+{{% choosable language typescript %}}
+
+### Strongly Typed Policy Documents
```

Comment:
Looking at the preview of this - it's a slightly  odd how this ends up looking with the whole section being based on a language chooser - in part because there is no language chooser above this on the page - so it isn't clear what causes it to appear or disappear. It is probably fine (assuming that most readers have already set a language preference elsewhere, and will be in the right "mode" already).  But an alternative choice could be to leave this section in - add [TypeScript only] to the section title - and then have the language chooser on the code block with only TS available?

Another thought, we may not even need to call this out in its own section.  This type is used throughout the other examples in the languages that support it.  That may be sufficient to make clear what this is and why?

Reply:
Agree. I went with your idea of just labeling the section "(TypeScript-only)" (and also making it an H3 so it doesn't show in the right hand ToC since it's about policy documents anyway). I didn't do the language switcher part because that has very strange effects on the page (it made it impossible to even select Python as the language elsewhere?)